skame/schemas/base.py

```diff
@@ -203,12 +203,18 @@ def validate(self, data: object) -> object:
 
 class Map(Schema):
     """Validator that validates a map of field names to validators."""
+    messages = {
```

Comment:
You should not define mutable attributes at class level, it is a very bad practice because all clases will share the same mutable dict. Instead define them in the constructor.